### PR TITLE
Work fe1 raulinn cleanup

### PR DESCRIPTION
**Pull request by RaulinN**

### CODE REVIEW

File: fe1-web/Model/Objects/keyPair.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+export class KeyPair {
+
+    public static publicKey: PublicKey;
+    public static privateKey: PrivateKey;
+
+    /**
+     * Initialize the keychain with a set of keys
+     */
+    public static initialise() {
```
Comment (pierluca) [inline fe1-web/Model/Objects/keyPair.ts:15]: I'd avoid creating a static class here.
It's convenient to have the possibility to represent any number of KeyPairs.

We could however create a separate "KeyStore" singleton, enabling us to retrieve the "default" key pair.

File: fe1-web/websockets/WebsocketApi.ts [hunk 2]
```diff
@@ -0,0 +1,266 @@
+import { eventTags, getCurrentLao, getCurrentTime, toString64 } from './WebsocketUtils';
```
Comment (pierluca) [inline fe1-web/websockets/WebsocketApi.ts:1]: I haven't had time to review this file yet

File: fe1-web/websockets/WebsocketApi.ts [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+import { ActionType, ObjectType } from '../Model/Network/Method/Message/data/messageData';
+import { Message } from '../Model/Network/Method/Message/message';
+import { JsonRpcRequest } from '../Model/Network/jsonRpcRequest';
+import { JsonRpcMethod } from '../Model/Network/jsonRpcMethods';
+import { Publish } from '../Model/Network/Method/publish';
+import { Hash } from "../Model/Objects/hash";
+import { PublicKey } from "../Model/Objects/publicKey";
+import { Timestamp } from "../Model/Objects/timestamp";
+import { Base64Data } from "../Model/Objects/base64";
+import { KeyPair } from "../Model/Objects/keyPair";
```
Comment (pierluca) [inline fe1-web/websockets/WebsocketApi.ts:12]: no need for `../` anymore :)

File: fe1-web/Model/Objects/hash.ts [hunk 4]
```diff
@@ -1,8 +1,33 @@
...
+        let str = '';
+        data.forEach((item) => { str += `"${Hash._escapeString(item)}",`; });
+        // remove the last comma and add square brackets around
+        str = `[${str.slice(0, -1)}]`;
+
+        const hash = sha256.create();
+
+        const bString = hash.update(str).array();
+        return Base64Data.encode(String.fromCharCode(...bString));
+    }
```
[the remark this answers is not in the document]
Comment (RaulinN) [reply fe1-web/Model/Objects/hash.ts:22]: Done! 

I checked on all 4 projects, and we all treat numbers the exact same way as strings. I do not know if we specified it somewhere at some point

**Pull request merged**